src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/_help.py

```diff
@@ -163,35 +169,34 @@
     type: command
     short-summary: List resources.
     examples:
-        - name: List all resource in a region.
+        - name: List all resource in the West US region.
           text: >
             az resource list --location westus
-        - name: List resource using a name.
+        - name: List a resource with the name 'resourceName'.
           text: >
-            az resource list --name thename
-        - name: List resources using a tag.
+            az resource list --name 'resourceName'
+        - name: List resources with the tag 'test'.
           text: >
-             az resource list --tag something
-        - name: List resources using a tag with a particular prefix.
+             az resource list --tag test
+        - name: List resources with a tag that starts with 'test'.
           text: >
-            az resource list --tag some*
-        - name: List resources using a tag value.
+            az resource list --tag test*
+        - name: List resources with the tag 'test' that have the value 'example'.
           text: >
-            az resource list --tag something=else
+            az resource list --tag test=example
 """
 
 helps['resource show'] = """
     type: command
     short-summary: Get information about a resource.
-    long-summary: For example /subscriptions/0000/resourceGroups/MyResourceGroup/providers/Microsoft.Provider/ResA/MyA/ResB/MyB/resC/MyC.
     examples:
-        - name: Show a virtual machine.
+        - name: Show a virtual machine resource named 'MyVm'.
           text: >
             az vm show -g MyResourceGroup -n MyVm --resource-type "Microsoft.Compute/virtualMachines"
         - name: Show a web app using a resource identifier.
           text: >
             az resource show --id /subscriptions/0b1f6471-1bf0-4dda-aec3-111111111111/resourceGroups/MyResourceGroup/providers/Microsoft.Web/sites/MyWebapp
-        - name: Show a subnet.
+        - name: Show a subnet in the 'microsoft.network' namespace which belongs to the virtual network 'MyVnet'.
```

Comment:
'Microsoft.Network'

Reply:
In this case, is the command case-sensitive? Should the argument also be renamed?

Reply:
The command isn't case sensitive, but in outputs, these are always capitalized.